cts/cts-cli.in

```diff
@@ -1267,7 +1267,7 @@ class CibadminRegressionTest(RegressionTest):
         """A list of Test instances to be run as part of this regression test."""
         basic_tests = [
             Test("Validate CIB", "cibadmin -Q", update_cib=True),
-            Test("Digest calculation", "cibadmin -Q | cibadmin -5 -p 2>&1"),
+            Test("Digest calculation", "cibadmin -Q | cibadmin -5 -p >/dev/null 2>&1"),
```

Comment:
:(

Let's leave the output, but cut out the hash, so we're at least testing the output format. Also comment the issue at the replacement so someone can try again in the future.